src/app/children/view-schools-component/view-schools.component.ts

```diff
@@ -0,0 +1,89 @@
+import {ChangeDetectorRef, Component, Input, OnChanges, OnInit, SimpleChanges, ViewChild} from '@angular/core';
+import {EditSchoolDialogComponent} from './edit-school-dialog/edit-school-dialog.component';
+import {EntityMapperService} from '../../entity/entity-mapper.service';
+import {MatDialog, MatTableDataSource, MatSort} from '@angular/material';
+import {Child} from '../child';
+import {LoggingService} from '../../logging/logging.service';
+import {ChildrenService} from '../children.service';
+import {SchoolWithRelation} from '../../schools/schoolWithRelation';
+
+@Component({
+  selector: 'app-view-schools-component',
+  templateUrl: './view-schools.component.html',
+  styleUrls: ['./view-schools.component.scss']
+})
+
+export class ViewSchoolsComponent implements OnInit, OnChanges {
```

Comment:
Could this use our `EntitySubrecordComponent`? That might avoid writing some of this code that's similar in other components.

Reply:
I will talk to @codingfabi and @christophscheuing what they think about it.

Reply:
Turns out @christophscheuing is already working on it (#207). But I think it would be good to merge these changes soon because I also edited this component.

Reply:
ah, nice. Maybe you guys can discuss directly which change should go in first and in which way.